common/djangoapps/course_modes/migrations/0005_auto_20151217_0958.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_modes', '0004_auto_20151113_1457'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Will this drop the field (and all existing data) before creating the field of the same name immediately after?

I just want to make sure we aren't going to lose any data